src/foam/dao/EasyDAO.js

```diff
@@ -279,8 +282,31 @@ return delegate;
         .setOf(getOf())
         .build();
       }
-      if ( getJournalType().equals(JournalType.SINGLE_JOURNAL) )
+      if ( getJournalType().equals(JournalType.SINGLE_JOURNAL) ) {
         return new foam.dao.java.JDAO(getX(), getOf(), getJournalName());
+      }
+      else if ( getJournalType().equals(JournalType.SHARED_JOURNAL) ) {
+
+        foam.dao.MDAO myMDAO = new foam.dao.MDAO(getOf());
+        X resourceStorageX = getX();
+        if ( System.getProperty("resource.journals.dir") != null ) {
+          resourceStorageX = getX().put(foam.nanos.fs.Storage.class,
+              new ResourceStorage(System.getProperty("resource.journals.dir")));
+        }
+        new foam.dao.FileJournal.Builder(resourceStorageX)
```

Comment:
If order doesn't matter, we could concatenate all the repository journals into a single journal shipped with the jar.